markpbaggett/primove_metadata
# Add Digital Collections mapping rules

## What does this Pull Request do?

This PR adds the mapping for Digital Collections for Primo VE as well as additional sample records.

## What's new?

This PR shares the normalization rules and regex needed to share our Digital Collections with Primo VE. The mapping reflects what is currently in production on Alma. Note that there is a hope to improve the creator and contributor mapping (currently all names are mapped to dc:creator) and that a Salesforce ticket has been opened for this. Given the fact that the current normalization rules have not changed in over a month, I felt it was important to go ahead and share what is presently being used instead of waiting any longer.

## How should this be tested?

Check to make sure the formatting is correct and the readthedocs site will build. 

## Interested parties

@markpbaggett
